whiteboard-api/src/main/proto/ai/lzy/v1/whiteboard/whiteboard.proto

```diff
@@ -25,36 +25,15 @@ message Whiteboard {
   }
```

Comment:
There is no filalized request and there is update request. So what does finalized status mean?

If it's still necessary for user but not for WhiteboardService, why is it enum, not just string?